parser/src/cfg/mod.rs

```diff
@@ -2989,9 +3029,21 @@ fn parse_sequences(exprs: &[&Vec<SExpr>], s: &ParserState) -> Result<KeySeqsToFK
         let mut subexprs = check_first_expr(expr.iter(), "defseq")?.peekable();
 
         while let Some(vkey_expr) = subexprs.next() {
-            let vkey = vkey_expr.atom(s.vars()).ok_or_else(|| {
-                anyhow_expr!(vkey_expr, "{SEQ_ERR}\nvirtual_key_name must not be a list")
-            })?;
+            let vkey = match vkey_expr {
+                SExpr::Atom(atom) => {
+                    s.lsp_hints
+                        .borrow_mut()
+                        .reference_locations
+                        .virtual_key
+                        .push_from_atom(atom);
+                    vkey_expr
+                        .atom(s.vars())
+                        .expect("must be atom since we're matching atom")
```

Comment:
Fly-by comment

The way this new code works isn't right because `match`ing an expr and finding an `Atom` might find you a `$var` that is actually a list.

You might want to add `span_atom` equivalent of `span_list`, which I haven't yet added because I didn't have the need for it.

Reply:
Agree. I've refactored in 99be6a2a8b98039e757c9f30fec8722b85c4b9b2. No need for `span_atom`